thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/AnomalyUtils.java

```diff
@@ -66,4 +66,22 @@ public static Boolean checkHasLabels(List<MergedAnomalyResultDTO> mergedAnomalyR
     }
     return false;
```

Comment:
Does the AnomalyFeedbackType contains 'NULL', what's the convention we used here?